### DISCUSSION TITLE
Config with IS and LA and all the other features, F1 Version

### DISCUSSION THREAD

**Discussion by Wassabichan** [mriscoc/Ender3V2S1#661] [Ideas]
i do have the weird bug that my printer isn't working when using self compiled versions of Marlin, so it would be a lifesaver if you could pack all the features in one file.

**Comment by mriscoc** [comment]
https://github.com/mriscoc/Special_Configurations

**Comment by BEprints** [comment]
https://github.com/mriscoc/Ender3V2S1/discussions/580
